### DISCUSSION TITLE
Change the default from auto_regenerate=True to auto_regenerate=False

### DISCUSSION THREAD

**Discussion by MartinThoma** [py-pdf/pypdf#2337] [Ideas]
Reading https://github.com/py-pdf/pypdf/pull/1945/files#diff-2376a2cb969cf3bd675e8f2a9623f3379907210fc6532ffd1e84b9cadae9b37d and https://github.com/py-pdf/pypdf/issues/1903#issuecomment-1603197095 it sounds to me as if the default should be `auto_regenerate=False`.

With the switch to pypdf==4.0.0 we could do that.

@pubpub-zz Did I understand that right? What do you think would be the best default for most users?

**Comment by pubpub-zz** [comment]
`Auto_Regenerate=False` should be better I think : it will preve the save dialog
